src/foundry/foundry.js/pixi/containers/placeableObjects/ambientLight.d.ts

```diff
@@ -1,117 +1,121 @@
-/**
- * An AmbientLight is an implementation of PlaceableObject which represents a dynamic light source within the Scene.
- *
- * @example
- * ```typescript
- * AmbientLight.create<AmbientLight>({
- *   t: "l",
- *   x: 1000,
- *   y: 1000,
- *   rotation: 0,
- *   dim: 30,
- *   bright: 15,
- *   angle: 360,
- *   tintColor: "#FF0000",
- *   tintAlpha: 0.05
- * });
- * ```
- */
-declare class AmbientLight extends PlaceableObject<AmbientLight.Data> {
+import { BaseAmbientLight } from '../../../../common/documents.mjs';
+
+declare global {
   /**
-   * A reference to the PointSource object which defines this light source area of effect
+   * An AmbientLight is an implementation of PlaceableObject which represents a dynamic light source within the Scene.
+   *
+   * @example
+   * ```typescript
+   * AmbientLight.create<AmbientLight>({
+   *   t: "l",
+   *   x: 1000,
+   *   y: 1000,
+   *   rotation: 0,
+   *   dim: 30,
+   *   bright: 15,
+   *   angle: 360,
+   *   tintColor: "#FF0000",
+   *   tintAlpha: 0.05
+   * });
+   * ```
    */
-  source: PointSource;
+  class AmbientLight extends PlaceableObject<BaseAmbientLight> {
+    /**
+     * A reference to the PointSource object which defines this light source area of effect
+     */
+    source: PointSource;
 
```

Comment:
controllIcon is missing (seems pretty weird in foundry though, what's going on there? :D
```js
  /**
   * A reference to the ControlIcon used to configure this light
   * @type {ControlIcon}
   */
  controlIcon
```

(yes, no semicolon, no initial value, no nothing...)